src/helpers/file.php

```diff
@@ -112,7 +112,11 @@ function append_storage_information(array $rows, array $params = [])
             return $rows;
         }
 
-        $fields =  get_files_fields($params['fields']);
+        if (empty($params['fields'])) {
+            $params['fields'] = '';
+        }
+
+        $fields =  get_files_fields($params['fields']) || '';
```

Comment:
I think we can use ternary operator here for better code quality.

` $fields =  !empty($params['fields']) ? get_files_fields($params['fields']) : '';`